pjsip/src/pjsip-ua/sip_inv.c

```diff
@@ -4082,8 +4082,8 @@ static pj_bool_t handle_uac_tsx_response(pjsip_inv_session *inv,
     if (inv->state != PJSIP_INV_STATE_DISCONNECTED &&
 	((tsx->status_code == PJSIP_SC_CALL_TSX_DOES_NOT_EXIST &&
 	    tsx->method.id != PJSIP_CANCEL_METHOD) ||
-	 tsx->status_code == PJSIP_SC_REQUEST_TIMEOUT ||
-	 tsx->status_code == PJSIP_SC_TSX_TIMEOUT))
```

Comment:
so PJSIP_SC_TSX_TIMEOUT is removed?

Reply:
They are the same, see [here](https://github.com/pjsip/pjproject/blob/master/pjsip/include/pjsip/sip_msg.h#L490), and as it is defined in a public header file, it should not be changed.